nodejs/provisioning/playbook.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - hosts: all
+  sudo: yes
```

Comment:
Note: this should be `become: yes` as per the 1.9.x convention (`sudo` is now deprecated in `2.x`).